src/main/java/com/fmi/findmeabuddy/serviceImpl/UserServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.fmi.findmeabuddy.serviceImpl;
+
+import com.fmi.findmeabuddy.Service.UserService;
+import com.fmi.findmeabuddy.dao.UserDao;
+import com.fmi.findmeabuddy.model.Account;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class UserServiceImpl implements UserService {
+    @Autowired
+    UserDao userDao;
+
+    @Override
+    public Account findUser(Long accountId) {
```

Comment:
1) We should find user by it's email not account id. That's what we're receiving on the login request.
2) Why do we need this additional method? Can't we use the `UserDao` object directly?